app/config/index.js

```diff
@@ -94,6 +97,9 @@ const config = {
   },
   endemics: {
     enabled: process.env.ENDEMICS_ENABLED
+  },
+  multiSpecies: {
+    enabled: process.env.MULTI_SPECIES_ENABLED
```

Comment:
should this, (and endemics above now it's required) be asserting that the value is 'true'? e.g.
```suggestion
    enabled: process.env.MULTI_SPECIES_ENABLED === 'true'
```

Reply:
good catch! thats my bad - this is why we have code reviews!

Reply:
I guess the endemics enabled also is wrong, and presumably has been for a while?

Reply:
This has opened a can of worms - there are loads of instances of this, across a few repos. It seems to be 'covered' by the schema having defaults, but it means that the env vars arent being used properly.